volatility/framework/plugins/windows/envars.py

```diff
@@ -0,0 +1,155 @@
+# This file is Copyright 2020 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+
+from typing import Callable, List, Generator, Iterable, Dict
+from volatility.framework import renderers, interfaces, objects, exceptions
+from volatility.framework.configuration import requirements
+from volatility.framework.objects import utility
+from volatility.plugins.windows import pslist
+from volatility.plugins.windows.registry import hivelist
+
+class Envars(interfaces.plugins.PluginInterface):
+    "Display process environment variables"
+
+    _version = (1, 0, 0)
+
+    @classmethod
+    def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]:
+        # Since we're calling the plugin, make sure we have the plugin's requirements
+        return [requirements.TranslationLayerRequirement(name = 'primary',
+                                                         description = 'Memory layer for the kernel',
+                                                         architectures = ["Intel32", "Intel64"]),
+                requirements.SymbolTableRequirement(name = "nt_symbols", description = "Windows kernel symbols"),
+                requirements.ListRequirement(name = 'pid',
+                                             description = 'Filter on specific process IDs',
+                                             element_type = int,
+                                             optional = True),
+                requirements.BooleanRequirement(name='SILENT',

```

Comment:
Looks good, I'm not sure SILENT needs to be all capitalized and might affect some UIs in the future so I'd recommend putting it to lowercase.  At some point we'll probably run `yapf` over this which will reformat it to include spaces around the equals signs in parameters, but that'll happen of its own accord.  5:)